tests/testFindAstrometryNetDataDir.py

```diff
@@ -29,7 +29,7 @@ def setupAstrometryNetDataDir(name, rootDir=None, verbose=False):
 class TestAstrometryNetDataDirDiscovery(unittest.TestCase):
 
     def test_photocal(self):
-        datapath = setupAstrometryNetDataDir('photocal')
+        datapath = setupAstrometryNetDataDir('t2155')
```

Comment:
This seems weird to me.  Why do you need a build tag to specify the a.net data directory?